content/agent/proxy.md

```diff
@@ -444,6 +444,97 @@ To verify that everything is working properly, review the HAProxy statistics at
 {{% /tab %}}
 {{< /tabs >}}
 
+## Using NGINX as a Proxy
+
+[NGINX][3] is a web server which can also be used as a reverse proxy, load balancer, mail proxy, and HTTP cache. You can use NGINX as a proxy to send logs to Datadog:
+
+`agent ---> nginx ---> Datadog` or  
+`external log shipper ---> nginx ---> Datadog`
+
+### Proxy forwarding with NGINX
+#### NGINX configuration
+
+This example `nginx.conf` can be used to proxy logs to the Datadog intake. It does TLS wrapping to ensure internal plaintext logs are encrypted between your proxy and Datadog's log intake API endpoint:
+
+```
+user nginx;
+worker_processes auto;
+error_log /var/log/nginx/error.log;
+pid /run/nginx.pid; 
+
+include /usr/share/nginx/modules/*.conf;
+events {
+    worker_connections 1024;
+}
+# TCP Proxy for Datadog logs
+stream {
+    server {
+        listen 10514; #proxy listen port
+        proxy_ssl on;
+        proxy_pass agent-intake.logs.datadoghq.com:10516;
+    }
+}
+```
+
+#### Shipper configuration
+
+{{< tabs >}}
+{{% tab "Agent v6" %}}
+
+To use the Datadog Agent as the logs collector, instruct the Agent to use the newly created proxy instead of establishing a connection directly with the logs intake by updating `datadog.yaml`:
+
+```yaml
+logs_config:
+  dd_url: myProxyServer.myDomain
```

Comment:
now the proper configuration is `logs_dd_url: myProxyServer.myDomain:10514` instead of using `dd_url` and `dd_port`